volplugin/handlers.go

```diff
@@ -168,6 +173,9 @@ func mount(master, host string) func(http.ResponseWriter, *http.Request) {
 			return
 		}
 
+		stopChan := addStopChan(uc.Request.Name)
+		go heartbeatMount(master, ttl, ut, stopChan)
```

Comment:
Oh, FTR, ceph will happily map rbd devices anywhere for N hosts, so we keep these locks for a very useful reason. :)